index.html

```diff
@@ -0,0 +1,218 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta name="description" content="Global Streaming is live-streaming website that allow users to record and broadcast video simultaneously.">
+
+    <!-- Font Awesome kit Code-------- -->
+    <script
+      src="https://kit.fontawesome.com/b21ce73c3c.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <!-- Google fonts ------- -->
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;900&display=swap" rel="stylesheet">
+
+    <!-- script links ----------- -->
+    <script src="./scripts/script.js" type="text/javascript" defer></script>
+    <script src="./scripts/static.js" type="text/javascript" defer></script>
+
+    <!--local  styles Link---------- -->
+    <link rel="icon" href="./images/logo.jpg" />
+    <link rel="stylesheet" href="./styles/styles.css" />
+    <title>Global streamimg</title>
+  </head>
+  <body>
+    <!-- page header section -->
+    <header>
+      <!-- top nav -->
+      <div class="top-nav">
+        <ul class="top-nav-links container">
+          <li>
+            <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a id="user-page1" href="#">My page</a></li>
+          <li><a class="sign-up-btn" id="sign-up-btn1" href="#">Sign Up</a></li>
+        </ul>
+      </div>
+      <!-- top nav ends here -->
+
+      <!-- mobile menu pop up -->
+      <div class="mobile-menu">
+        <ul class="menu-list">
+          <li><a class="mobile-link" href="./index.html">Home</a></li>
+          <li><a class="mobile-link" href="./about.html">About</a></li>
+          <li><a class="mobile-link" href="#event-categories">Event</a></li>
+          <li><a class="mobile-link" href="#">Live Streaming</a></li>
+          <li><a class="mobile-link" class="sign-up" id="sign-up-btn2" href="#">Sign Up</a></li>
+        </ul>
+      </div>
```

Comment:
- [ Optional ] For user interaction I recommended you adjust hamburger menu on the left side instead of right side.
![Capture](https://user-images.githubusercontent.com/93598189/196743955-c9c0dfbc-f695-4472-adde-a1c7dccf2f47.PNG)

Reply:
Hello @Abenezer-Tilahun, thanks for your observation updates have been made.